R/get_it_extract_paths.R

```diff
@@ -1,13 +1,14 @@
-#' Get the full path to the IT
-#' Long Term Conditions extract
+#' IT Long Term Conditions
```

Comment:
I think these types need to have 'file path' in the title for clarity.

```suggestion
#' IT Long Term Conditions file path
```